apps/communications/contacts/js/views/settings.js

```diff
@@ -713,6 +716,7 @@ contacts.Settings = (function() {
     newOrderByLastName = !orderCheckBox.checked;
     utils.cookie.update({order: newOrderByLastName});
     updateOrderingUI();
+    Cache.evict();
```

Comment:
Should make sense that we evict before `updateOrderingUI` and rebuild when list is done?